File: java/Program/ProdukElektronik.java
// =============================
// Class ProdukElektronik
// Level 1 (Parent class)
// =============================

public class ProdukElektronik {
    // =============================
    // Atribut dasar produk elektronik
    // =============================
    private String namaProduk; // menyimpan nama produk
    private String merek;      // menyimpan merek produk
    private int harga;         // menyimpan harga produk

    // =============================
    // Constructor kosong (default)
    // =============================
    public ProdukElektronik() {}

    // =============================
    // Constructor dengan parameter
    // Digunakan untuk menginisialisasi data produk langsung
    // =============================
    public ProdukElektronik(String namaProduk, String merek, int harga) {
        this.namaProduk = namaProduk;
        this.merek = merek;
        this.harga = harga;
    }

    // =============================
    // Getter dan Setter
    // Untuk mengakses dan mengubah atribut
    // =============================
    public String getNamaProduk() { return namaProduk; }
    public String getMerek() { return merek; }
    public int getHarga() { return harga; }

    public void setNamaProduk(String namaProduk) { this.namaProduk = namaProduk; }
    public void setMerek(String merek) { this.merek = merek; }
    public void setHarga(int harga) { this.harga = harga; }
}

File: python/Program/ProdukElektronik.py
# =============================
# Class ProdukElektronik
# Level 1 (Parent class)
# =============================
class ProdukElektronik:
    # Constructor
    def __init__(self, nama_produk="", merek="", harga=0):
        """
        Inisialisasi atribut dasar produk elektronik
        :param nama_produk: nama produk
        :param merek: merek produk
        :param harga: harga produk
        """
        self.nama_produk = nama_produk
        self.merek = merek
        self.harga = harga

    # =============================
    # Getter dan Setter
    # =============================
    def get_nama_produk(self):
        return self.nama_produk

    def set_nama_produk(self, nama_produk):
        self.nama_produk = nama_produk

    def get_merek(self):
        return self.merek

    def set_merek(self, merek):
        self.merek = merek

    def get_harga(self):
        return self.harga

    def set_harga(self, harga):
        self.harga = harga